bin/local-env/docker-compose.yml

```
@@ -0,0 +1,41 @@
+version: '3.1'
+
+services:
+  wordpress:
+    image: wordpress
+    ports:
+      - "127.0.0.1:8899:80"
+    environment:
+      WORDPRESS_DB_HOST: mysql
+      WORDPRESS_DB_PASSWORD: example
+      ABSPATH: /usr/src/wordpress/
+      WORDPRESS_DEBUG: 1
+      WORDPRESS_CONFIG_EXTRA: |
+        define( 'SCRIPT_DEBUG', true );
+        define( 'FS_METHOD', 'direct' );
+    volumes:
+      - wordpress_data:/var/www/html
+      - ../../:/var/www/html/wp-content/plugins/web-stories
+    depends_on:
+      - mysql
+
+  cli:
+    image: wordpress:cli
+    user: xfs
+    volumes:
+      - wordpress_data:/var/www/html
+      - ../../:/var/www/html/wp-content/plugins/web-stories
+    depends_on:
+      - mysql
+      - wordpress
+    command: tail -f /dev/null
+
+  mysql:
+    image: mysql:5.7
+    restart: always
```

```suggestion
    restart: always
    volumes:
      - ./data/mysql:/var/lib/mysql
```

This tweak will keep database, so you don't have to recreate stories everytime you resetart your container.

---

data/mysql will also need to be added to the .gitignore.

---

Done in 8608ac4bd50767f904632e113a368f177ae3971a